### ISSUE TITLE
wrapwrite doesn't encode output

### ISSUE THREAD

**Issue by stefanor** [aaronsw/html2text#4]
```
$ python html2text.py http://google.com/
Traceback (most recent call last):
  File "html2text.py", line 473, in <module>
    wrapwrite(html2text(data, baseurl))
  File "html2text.py", line 436, in wrapwrite
    def wrapwrite(text): sys.stdout.write(text)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xbb' in position 85: ordinal not in range(128)
```

This is not an issue with Python3.

**Comment by aaronsw** [comment]
This is an issue for me in Python3 as well, but I couldn't figure out how to implement a solution that worked in both 2 and 3.

**Comment by stefanor** [comment]
Aah, if I set LANG to a non-UTF-8 locale (such as C) then it'll fail with python3.

I've got a proposed solution in my encode_output branch. Pull request incoming...

**Comment by aaronsw** [comment]
Fixed in 6c72d90.
